commit 104285595577d6b09e944a51450ae7d4643e6e52 (refs/changes/69/4569/1)
Author: Rui Loura <rui@joyent.com>
Date:   2018-07-25T18:23:23+00:00 (1 year, 2 months ago)
    
    TRITON-592 when provisioning vmapi should check for network pools before networks (bump version)

diff --git a/package.json b/package.json
index c17bdd7..77c4a3a 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "vmapi",
   "description": "VMs API",
-  "version": "9.5.3",
+  "version": "9.6.0",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
